src/actors/rqgActorSheet.ts

```diff
@@ -1062,6 +1062,26 @@ export class RqgActorSheet extends ActorSheet<
           }
         });
       });
+
+    // Add Passion button
+    htmlElement?.querySelectorAll<HTMLElement>("[data-passion-add]").forEach((el) => {
+      el.addEventListener("click", async () => {
+        console.log("CLICK");
+        const defaultItemIconSettings: any = getGame().settings.get(
+          systemId,
+          "defaultItemIconSettings"
+        );
+        const newPassionName = localize("RQG.Actor.Passion.NewPassionName");
+        let passion = {
+          name: newPassionName,
```

Comment:
It would be nice to not need another translation just for this. Maybe one could pick one passion as a default and the player can then change it to what they want in the item sheet. Does "Loyalty" seems like a good default, feels like it's among the most common to add.

Reply:
Done